docs/rpc-endpoints.md

```diff
@@ -343,4 +343,12 @@ object of the following form:
 
 Determine whether a given trait is implemented within the specified contract (either explicitly or implicitly).
 
-See OpenAPI [spec](./rpc/openapi.yaml) for details.
\ No newline at end of file
+See OpenAPI [spec](./rpc/openapi.yaml) for details.
+
+### GET /v2/health
+
+Determine whether a node is healthy. The notion of health used for this 
```

Comment:
Where did you get this operationalization for "healthy"?

I see https://github.com/blockstack/stacks-blockchain/issues/1229. Is this an analogy to "v1 health"?

Reply:
Yes - discussed this notion of "health" here with Aaron (https://discord.com/channels/621759717756370964/730814571517968436/837017500763422817)

Reply:
Sounds good.. only comment I had below was what if your "is_healthy" is rewritten to something like "matches_peers", leaving open the possibility of adding multiple "health"  metrics to this endpoint over time?

Reply:
I think we can add more notions of health to this same endpoint as the need arises

Reply:
yes! as we find more things we want to know, it's easy within this structure to add a new k:v